src/installer/snaps/dotnet-runtime-3.0/snap/snapcraft.yaml

```diff
@@ -0,0 +1,45 @@
+name: dotnet-runtime-30
```

Comment:
3.0 is out of support. Do we need to keep it?

Reply:
> 3.0 is out of support. Do we need to keep it?

@leecow do you know if we need 3.0?

Reply:
No need to keep 3.0.

Reply:
I've removed the 3.0 snap.